site/src/components/Tooltips/ResourcesHelpTooltip.tsx

```diff
@@ -0,0 +1,27 @@
+import {
+  HelpTooltip,
+  HelpTooltipLink,
+  HelpTooltipLinksGroup,
+  HelpTooltipText,
+  HelpTooltipTitle,
+} from "./HelpTooltip/HelpTooltip"
+
+export const Language = {
+  resourceTooltipTitle: "What is a resource?",
+  resourceTooltipText: "A resource is an infrastructure object that is create when the workspace is provisioned.",
```

Comment:
```suggestion
  resourceTooltipText: "A resource is an infrastructure object that is create when the workspace is provisioned.",
```

👍🏽 clear and concise